src/database/sequelize.ts

```diff
@@ -0,0 +1,17 @@
+import {databaseName, password, userName} from '../config/config';
+import {Sequelize} from 'sequelize';
+import userModel from '../database/user.model';
+ 
+const sequelize = new Sequelize(databaseName, userName, password, {
+  dialect: 'postgres',
+  host: 'localhost'
+});
+
+const User = sequelize.define("user", userModel, {
+  timestamps: false

```

Comment:
I think you should use timestamps, is it about created_at and updated_at?

Reply:
You are right, It is about created_at and updated_at
Do we really need that? This option create additional column "createdAt" and I need recreate table, change models and so on

Reply:
I think so, you should use migrations if you need changes in db you can create additional migration or if you haven't deployed yet your app you can just roll back your last migration, fix it and run it again with changes. Could you also add script to create users in db, I didn't see it.

Reply:
Added timestamps, migrations, seeders